.github/workflows/greetings.yml

```
@@ -25,13 +25,13 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           issue-message: >
-            👋 Hello @{{ sender.login }}!  
+            👋 Hello @{{ github.event.issue.user.login  }}!  
             Thank you for opening this issue. We appreciate your contribution to the project!  
             Please review our [contribution guidelines](https://github.com/rahulc0dy/micro-auth/blob/main/CONTRIBUTING.md) if you haven’t already.  
             A team member will respond soon! 🚀
 
           pr-message: >
-            👋 Hi @{{ sender.login }}!  
+            👋 Hi @{{ github.event.issue.user.login }}!  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)